cluster-provision/base/scripts/vm.sh

```diff
@@ -87,4 +87,5 @@ exec qemu-system-x86_64 -enable-kvm -drive format=qcow2,file=${next},if=virtio,c
   -device virtio-net-pci,netdev=network0,mac=52:55:00:d1:55:${n} \
   -netdev tap,id=network0,ifname=tap${n},script=no,downscript=no \
   -device virtio-rng-pci \
-  -vnc :${n} -cpu host -m ${MEMORY} -smp ${CPU} ${QEMU_ARGS}
+  -vnc :${n} -cpu host -m ${MEMORY} -smp ${CPU} ${QEMU_ARGS} \
+  -serial pty
```

Comment:
Please add a newline

Reply:
consider that gocli already adds it i think
look on the ps -ef in the container

Reply:
Newline in the script, it has nothing to do with gocli. Every line should be finished with `\n`, thus the red mark here.

Reply:
i mean that gocli already adds -serial imo
cluster-provision/gocli/cmd/run.go:     qemuArgs += " -serial pty"

Reply:
I'm not commenting on the functionality. I comment on the fact that you don't finish the file with a newfile, which is consider good manners while handling source files.